modules/wallet/consts.go

```diff
@@ -46,14 +46,9 @@ func dustValue() types.Currency {
 
 // defragFee is the miner fee paid to miners when performing a defrag
 // transaction.
-//
-// TODO: These need to be functions of the wallet that interact with the
-// transaction pool.
-func defragFee() types.Currency {
-	// 35 outputs at an estimated 250 bytes needed per output means about a 10kb
-	// total transaction, much larger than your average transaction. So you need
-	// a lot of fees.
-	return types.SiacoinPrecision.Mul64(10)
+func (w *Wallet) defragFee(txnSize uint64) types.Currency {
+	minFee, _ := w.tpool.FeeEstimation()
+	return minFee.Mul64(txnSize)
 }
```

Comment:
can delete this now that it's not used anywhere

Reply:
The function as it stands violates locking convention, which is that unexported functions of a module need to be called under lock. But if you do that, this function calls out to another module, which is a deadlock risk. Needs to be named managedDefragFee at least.

But this code is identical to all code that calculates fees, I don't see why it needs a special helper function. I believe this function has been made obsolete, and the code should just be inlined everywhere.